src/geometry/__tests__/resize.test.ts

```diff
@@ -1,25 +1,43 @@
-import { encodePng } from '../../save';
+import path from 'node:path';
 
-test.skip('compare result of resize with opencv (nearest)', () => {
+import { encodePng, write } from '../../save';
+
+test('compare result of resize with opencv (nearest)', async () => {
   const img = testUtils.load('opencv/test.png');
+  const expectedImg = testUtils.load('opencv/testResizeNearest.png');
 
   const resized = img.resize({
     xFactor: 10,
     yFactor: 10,
     interpolationType: 'nearest',
   });
 
+  const substraction = expectedImg.clone().subtract(resized);
```

Comment:
Sorry it seems I forgot to submit the comment on my previous review: please remove the debugging code.